### DISCUSSION TITLE
Rounded corner

### DISCUSSION THREAD

**Discussion by Dmitriy147** [fyne-io/fyne#4501] [General]
In the last version v2.4.3, building in win10, i can see only button rounded corners - not rounded main form corners.. In the github page i can see two screenshots with rounded corners of the main window. How could i setup such rounded form's corners?

**Comment by Jacalz** [comment]
What do you mean with "main form"? If you are talking about rounded corners on application windows, then that's highly dependent on your operating system. macOS does that and maybe Windows 11 but most other systems do not.
